jmh-tests/src/main/java/org/eclipse/collections/impl/jmh/IntIntMapSmallStressTest.java

```diff
@@ -353,7 +353,7 @@ public void kolobokeRemove()
             {
                 newMap.remove(this.kolobokeIntKeysForMap[i]);
             }
-            if (newMap.size() != 0)
+            if (!newMap.isEmpty())
```

Comment:
`notEmpty()`?

Reply:
newMap is of type IntIntMap. IntIntMap interface does not have a notEmpty() method.